gui/mass-remove.lua

```diff
@@ -0,0 +1,314 @@
+--building/construction mass removal/suspension tool
+
+--[====[
+
+gui/mass-remove
+===============
+Allows removal of buildings/constructions and suspend/unsuspend using
+a box selection.
+
+The following marking modes are available.
+
+- Suspend (s): suspends the construction of a planned building/construction
+- Unsuspend (p): resumes the construction of a planned building/construction
```

Comment:
perhaps a slightly more mnemonic lettering would be to use lower case for one action and the upper case of that letter for the corresponding "un" action. e.g. Suspend (s), Unsuspend (S), Remove Construction (n), Unremove Construction (N), Remove Building (x), Unremove Building (X), Remove All (a), Unremove All (A)

Reply:
See, that's the sorta stuff that I won't come up with by myself because I'm not that good at UX. Love it, I'll change the keybindings real fast, very intuitive.

Reply:
Applied in 480f35b